cfgov/unprocessed/js/modules/util/dom-traverse.js

```diff
@@ -1,5 +1,19 @@
 'use strict';
 
+var typeCheckers = require( './type-checkers' );
+
+/**
+ * Queries for the first match unless an HTMLNode is passed
+ * @param   {object|string} expr HTMLNode or string to query for
```

Comment:
Minor: should be `{(HTMLNode|string)}` (with parens). This is wrong in our existing codebase, but it should have the parens around it according to http://usejsdoc.org/tags-type.html